### PR TITLE
Add skeleton rules & firebase config

### PR DESCRIPTION
**Pull request by pradigunara**

### CODE REVIEW

File: src/util/rules.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  [HANDS.STRAIGHT_FLUSH](cards) {},
+}
+
+const tieBreaker = {
+  [HANDS.TWO_PAIR](self, other) {},
+  [HANDS.STRAIGHT](self, other) {},
+  [HANDS.FLUSH](self, other) {},
+  [HANDS.FULL_HOUSE](self, other) {},
+  [HANDS.FOUR_OF_A_KIND](self, other) {},
+  [HANDS.STRAIGHT_FLUSH](self, other) {},
```
Comment (pradigunara) [inline src/util/rules.js:19]: please implement @andreasanggaraa

File: src/constant/hands.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash'
+
+/**
+ * @typedef {String} HandsEnum
+ * @enum {HandsEnum}
+ */
+export const HANDS = {
+  TWO_PAIR: 'TWO_PAIR',
```
Comment (andreasanggaraa) [inline src/constant/hands.js:8]: ini two pair maksudnya 2 pair + sampah?

Comment (pradigunara) [reply src/constant/hands.js:8]: hooh bikin dulu aja, nanti mau enable apa engga belakangan dah

Comment (andreasanggaraa) [reply src/constant/hands.js:8]: noted

Comment (pradigunara) [reply src/constant/hands.js:8]: four of kind nya jg pake sampahan

**Pull request merged**